cmd/deploy/endpoints.go

```diff
@@ -227,7 +227,7 @@ func (dc *EndpointGetter) showEndpoints(ctx context.Context, opts *EndpointsOpti
 		}
 	default:
 		if len(eps) == 0 {
-			oktetoLog.Printf("There are no available endpoints for '%s'\n", opts.Name)
+			oktetoLog.Information("There are no available endpoints for '%s'.\n   Follow this link to know more about how to create public endpoints for your application:\n   https://www.okteto.com/docs/cloud/ssl/", opts.Name)
```

Comment:
@nitishfy in the screenshot, `There are...` and `Follow ...` aren't aligned, I think you need one more space before `Follow...` and the link

Reply:
Resolved!